File: csharp-dotnet2/src/main/CsharpDotNet2/IO/Swagger/Model/V3ZoneInfo.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class V3ZoneInfo {
    /// <summary>
    /// Gets or Sets MinZone
    /// </summary>
    [DataMember(Name="MinZone", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "MinZone")]
    public int? MinZone { get; set; }

    /// <summary>
    /// Gets or Sets MaxZone
    /// </summary>
    [DataMember(Name="MaxZone", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "MaxZone")]
    public int? MaxZone { get; set; }

    /// <summary>
    /// Gets or Sets UniqueZones
    /// </summary>
    [DataMember(Name="UniqueZones", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "UniqueZones")]
    public List<int?> UniqueZones { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class V3ZoneInfo {\n");
      sb.Append("  MinZone: ").Append(MinZone).Append("\n");
      sb.Append("  MaxZone: ").Append(MaxZone).Append("\n");
      sb.Append("  UniqueZones: ").Append(UniqueZones).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: dart/lib/model/v3_zone_info.dart
part of swagger.api;

class V3ZoneInfo {
  
  int minZone = null;

  int maxZone = null;

  List<int> uniqueZones = [];

  V3ZoneInfo();

  @override
  String toString() {
    return 'V3ZoneInfo[minZone=$minZone, maxZone=$maxZone, uniqueZones=$uniqueZones, ]';
  }

  V3ZoneInfo.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    minZone = json['MinZone'];
    maxZone = json['MaxZone'];
    uniqueZones = (json['UniqueZones'] as List).map((item) => item as int).toList();
  }

  Map<String, dynamic> toJson() {
    return {
      'MinZone': minZone,
      'MaxZone': maxZone,
      'UniqueZones': uniqueZones
     };
  }

  static List<V3ZoneInfo> listFromJson(List<dynamic> json) {
    return json == null ? new List<V3ZoneInfo>() : json.map((value) => new V3ZoneInfo.fromJson(value)).toList();
  }

  static Map<String, V3ZoneInfo> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, V3ZoneInfo>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new V3ZoneInfo.fromJson(value));
    }
    return map;
  }
}
